l10n_br_fiscal/views/product_product_view.xml

```diff
@@ -85,7 +94,11 @@
                             </field>
                         </group>
                         <group>
-                            <field name="fiscal_genre_id" required="1" />
+                            <field
+                                name="fiscal_genre_id"
+                                required="1"
+                                groups="l10n_br_fiscal.group_manager"
+                            />
```

Comment:
@renatonlima @marcelsavegnago I think this group is not quite right. How could we fix it?

Reply:
@rvalyi Using `attrs`...